lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -179,14 +179,17 @@ class UnusedCSSRules extends Audit {
       return UnusedCSSRules.mapSheetToResult(indexedSheets[sheetId], pageUrl);
     }).filter(sheet => sheet && sheet.wastedBytes > 1024);
 
+
+    const headings = [
+      {key: 'url', itemType: 'url', text: 'URL'},
```

Comment:
to have an individual row change the itemType? that seems a little weird. whyd we want diff itemtypes in the column?